k8s/base/gridexplore-app-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: gridexplore-app-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/service-upstream: "true"
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+    # springfox swagger-ui uses X-Forwarded-Prefix to find the
+    # base url when behind a reverse proxy.
+    # nginx-ingress-controller V0.24 has a simple alternative
+    # but latest minikube 1.1.0 uses V0.23 (untested):
+    # nginx.ingress.kubernetes.io/x-forwarded-prefix: "/gridexplore"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      rewrite ^/gridexplore$ /gridexplore/ permanent;
+      proxy_set_header X-Forwarded-Prefix         /gridexplore/;
+    nginx.ingress.kubernetes.io/server-snippet: |
+      explore_slashes off;
```

Comment:
merge_slashes instead ?

Reply:
yeah i should verify automatic replacements.